src/libostree/ostree-sysroot-cleanup.c

```diff
@@ -353,34 +354,18 @@ cleanup_ref_prefix (OstreeRepo         *repo,
                     GCancellable       *cancellable,
                     GError            **error)
 {
```

Comment:
Maybe a small note at the top of this function that we're called assuming an active transaction is underway?